### PR TITLE
PlayerESP for Wins and Rank + Additionally Misc Feature to Enable SniperCrosshair in Scope mode optional

### PR DESCRIPTION
**Pull request by 0xE232FE**
As requested:
https://github.com/danielkrupinski/Osiris/issues/668#issue-506718365

But I have problem with the NETVAR Functions getWins() and getRank()

Can someone help me here please?

### CODE REVIEW

File: Osiris/Hacks/Esp.cpp [hunk 1]
```diff
@@ -231,6 +231,12 @@ static void renderPlayerBox(Entity* entity, const Config::Esp::Player& config) n
 
         float drawPositionY = bbox.bottom;
 
+        if(config.competitiveRank)
+			renderPositionedText(config.font, (std::to_wstring(entity->getRank() ) + L" Rank").c_str(), config.competitiveWinsColor, { bbox.x1 + 5, drawPositionY });
```
Comment (0xE232FE) [inline Osiris/Hacks/Esp.cpp]: entity->getRank() doesn't show the rank as int. Please help me with this

File: Osiris/Hacks/Esp.cpp [hunk 2]
```diff
@@ -231,6 +231,12 @@ static void renderPlayerBox(Entity* entity, const Config::Esp::Player& config) n
...
+
+		if (config.competitiveWins)
+			renderPositionedText(config.font, (std::to_wstring(entity->getWins() ) + L" Wins").c_str(), config.competitiveWinsColor, { bbox.x1 + 5, drawPositionY });
```
Comment (0xE232FE) [inline Osiris/Hacks/Esp.cpp]: entity->getWins() doesn't show the rank as int. Please help me with this

**Comment by 0xE232FE** [comment]
I added the
Misc Option for SniperCrosshair if you turn it on and want additionally have SniperCrosshair in Scoped Mode you can turn ignor scope Crosshair on.

**Comment by xsincbt** [comment]
Nice job that is actually pretty smart

**Comment by Bywnb** [comment]
G ood job! Daniel plz pull it @danielkrupinski

**Comment by xsincbt** [comment]
just got to fix alittle

**Pull request closed without merging**